app/helpers/summary_helper.rb

```diff
@@ -44,13 +44,15 @@ def build_data_row(object, field)
   end
 
   def value_style(value)
-    ['column-two-thirds',
-     (
-      {
-        '✓' => ' summary-result passed',
-        '✗' => ' summary-result failed',
-        'W' => ' summary-result part'
-      }[value.to_s.first] || '')
-    ].join
+    styles = ['column-two-thirds']
+    case value
+    when /^✓/
+      styles << 'summary-result passed'
+    when /^x/
+      styles << 'summary-result failed'
+    when /^Waiting for/
+      styles << 'summary-result part'
+    end
+    styles
```

Comment:
And actually thinking about it, maybe you need a `.join(' ')` in the end, before you return, to make it a string?